shared/naturalcrit/markdown.js

```diff
@@ -902,7 +946,13 @@ let globalPageNumber = 0;
 const Markdown = {
 	marked : Marked,
 	render : (rawBrewText, pageNumber=0)=>{
-		globalVarsList[pageNumber] = {};					//Reset global links for current page, to ensure values are parsed in order
+		const lastPageNumber = pageNumber > 0 ? globalVarsList[pageNumber - 1].HB_PageNumber.content : 0;
+		globalVarsList[pageNumber] = {							//Reset global links for current page, to ensure values are parsed in order
+			'HB_PageNumber' : {									//Add document variables for this page
+				content  : !isNaN(Number(lastPageNumber)) ? Number(lastPageNumber) + 1 : lastPageNumber,
```

Comment:
Are we doing this this way to allow the user to reset the value  arbitrarily?

Reply:
Correct, this is to allow the user to set the page number, and have that affect future pages as well. If they set it to something other than a number, then that is also carried forward, but without being changed.